subsys/bluetooth/controller/ll_sw/lll.h

```diff
@@ -327,21 +327,24 @@ struct node_rx_ftr {
 #endif /* CONFIG_BT_CTLR_EXT_SCAN_FP */
 
 #if defined(CONFIG_BT_CTLR_ADV_EXT) && defined(CONFIG_BT_OBSERVER)
+	uint8_t  phy_flags:1;
+	uint8_t  scan_req:1;
+	uint8_t  scan_rsp:1;
+
 #if defined(CONFIG_BT_CTLR_PRIVACY)
 	uint8_t  direct_resolved:1;
 #endif /* CONFIG_BT_CTLR_PRIVACY */
 
-	uint8_t  aux_lll_sched:1;
-	uint8_t  aux_w4next:1;
-	uint8_t  aux_failed:1;
 #if defined(CONFIG_BT_CTLR_SYNC_PERIODIC)
 	uint8_t  sync_status:2;
 	uint8_t  sync_rx_enabled:1;
 #endif /* CONFIG_BT_CTLR_SYNC_PERIODIC */
 
-	uint8_t  phy_flags:1;
-	uint8_t  scan_req:1;
-	uint8_t  scan_rsp:1;
+	uint8_t  aux_sched:1;
+	uint8_t  aux_lll_sched:1;
+	uint8_t  aux_failed:1;
+
+	uint16_t aux_data_len;
```

Comment:
Name of the member suggests that is will hold extended advertising data length.
In diff we can see that also periodic sync data are stored in this member. 
So, name may misled one while reading the code if I'm correct in my understanding.

Reply:
Ok, I thought that u use `aux_` in context of extended advertising with exclusion of periodic advertising.